app/controllers/v0/institution_programs_controller.rb

```diff
@@ -116,5 +116,11 @@ def add_active_search_facets(raw_facets)
       add_country_search_facet(raw_facets)
       raw_facets
     end
+
```

Comment:
Can we dry this code up and move the page method to a shared helper?

Reply:
moved function out to ApiController parent class